acrg/paris_formatting/make_paris_outputs.py

```diff
@@ -115,24 +115,33 @@ def make_country_output(
         "NW_EU_CONTINENT": "BEL-DEU-FRA-LUX-NLD",
     }
 
-    region_traces = []
-    for region, countries_str in regions_dict.items():
-        region_countries = countries_str.split("-")
-        region_ds = (
-            country_traces_merged.sel(country=region_countries).sum("country").expand_dims({"country": [region]})
+    try:
+        region_traces = []
+        for region, countries_str in regions_dict.items():
+            region_countries = countries_str.split("-")
+            region_ds = (
+                country_traces_merged.sel(country=region_countries).sum("country").expand_dims({"country": [region]})
+            )
+            region_traces.append(region_ds)
+
+        region_traces_merged = xr.concat(region_traces, dim="country")
+
+        # combine country and region traces
+        all_traces_merged = xr.merge([country_traces_merged, region_traces_merged])
+
+        country_output = xr.merge(
```

Comment:
This code can come after the try/except/else blocks. In the except block, define `all_traces_merged = country_traces_merged`, then the calculate stats code is the same, no matter if the exception was found or not.